test/blackbox-tests/test-cases/ctypes/exe-vendored.t/run.t

```diff
@@ -0,0 +1,2 @@
+  $ dune exec ./example.exe
```

Comment:
This test needs a brief description of what feature is being tested and the setup used.